documentation_template_latex/zubaxdoc.cls

```diff
@@ -255,7 +256,7 @@
 
 % Use this environment to define a table within the wrapper environment defined above.
 \newenvironment{ZubaxWrappedTable}[2][\empty]{
-    \NoLeftSkip
+%    \NoLeftSkip
```

Comment:
After some consideration and the evaluation of our existing documents, I am inclined to leave NoLeftSkip as a default behavior because many tables that we use require more width than is available with the left skip in place. Maybe we should instead add an optional parameter to `ZubaxWrappedTable` and `ZubaxTableWrapper` that would insert the skip (and reduce the table width accordingly) when specified?

The practice of using variable left skip is quite common in technical documentation. As an example, I am attaching here the datasheet for an ST Microelectronics chip: [stm32f446.pdf](https://github.com/Zubax/document_templates/files/4721796/stm32f446.pdf). You can see that wide tables and large figures are positioned without the skip.

Reply:
Dare I say, this is probably because these documents were typeset by an engineer rather than a graphic designer. In the ST Microelectronics document, for example, some tables and figures are flush against the left margin (e.g. table 2, figure 3) whereas other are not (e.g. table 3, figure 2). From a typographical standpoint, this is a bit of a mess (read [this document](https://www.scribd.com/document/63081645/Typography-janson-Syntax-107514) on best practices). However, if this is how electrical engineers like to read their documents, then we should stick to the standard industry practices.

Earlier, I asked how much of the document could be changed and I was told that nearly all of it could be. However, somebody needs to control the design (look & feel) of the documents. Normally, this would be worked out between the graphic designer and marketing teams. In our case, should this be me, the author, or someone else in Zubax?

Reply:
Another design point in the zubaxdoc class is the decision to put table captions above the table and the figure captions below the figure. The class cites [this post](https://tex.stackexchange.com/questions/3243/why-should-a-table-caption-be-placed-above-the-table) as the reason.

In my opinion, this is a bit dubious. I agree with the analysis of how English readers process information but not with the proposed solution. By putting the caption above the table, you effectively make it a heading. If you want a heading on a table, I would use a _heading_ style; that is what it is there for. To me, a caption is simple a reference marker should the reader need to find the table or figure from the table of contents.

That said, there are good examples where captions are used as heading.  If you make this design choice, I would suggest being consistent throughout the document. I notice in the ST Microelectronics document, for example, that both table and figure captions are placed above in both cases.

Reply:
**For the document designer** : A reminder about bullet point labels. Are we sticking to dot bullets for all levels, or should they change for each level? I notice the example ST Microelectronics document uses the latter format whereas we use the former.